crates/host/src/wasmbus/mod.rs

```diff
@@ -2380,7 +2381,7 @@ impl Host {
         claims: &jwt::Claims<jwt::Actor>,
         annotations: &Annotations,
         actor_ref: impl AsRef<str>,
-        max: Option<NonZeroUsize>,
+        max: NonZeroUsize,
```

Comment:
I guess there's no use case for forcing actor instantiation but having no max? I might have thought that would be a case for some sort of "preloading"

Also, did we miss `max_instances` rename here?

Reply:
Yeah this was just shorthand, I can go refactor for clarity, no need to have ambiguity.

What I was thinking is that if we forced instantiation with no max, it would only ever be different if there were more than 2^32 concurrently executing instances of a single component. The benefit of getting rid of the option is clarity, no need to speculate between the difference of `None` and `Some(0)`

Reply:
ah OK -- yeah that's definitely a simplification, and I guess if we ever want to have some sort of preload operation we can add one explicitly